### ISSUE TITLE
Error due to tuple-constructor functions.

### ISSUE THREAD

**Issue by xldenis** [xldenis/creusot#414]
Actually attempting to use a tuple constructor will trigger an error in creusot.

**Comment by xldenis** [comment]
This no longer _crashes_, we do give an error about unsupported types of constants but I think this is acceptable.

**Comment by jhjourdan** [comment]
What's a tuple constructor in this context? I'm pretty sure `(1, 2)` does not crash in Creusot.

**Comment by jhjourdan** [comment]
Yeah, well, in general we should add the necessary machinery to automatically implement the logical closure traits for function pointer, but this is acceptable not to do this for now.
